shopelectro/management/commands/price.py

```diff
@@ -33,16 +43,82 @@ def create(self):
             file.create()
 
 
-class Price:
-    pass
+class Context(newcontext.Context):
+    """DB data, extracted for price file."""
+
+    def __init__(self, target: str):
+        self.target = target
+
+    def context(self) -> dict:
+        categories = CategoriesFilter(self.target).qs()
```

Comment:
@artemiy312 , most of the code of current `TargetCategories` do filtering. Including utm filtering, but it's filtering.
And ignored categories filtering is out of the target bounds

Reply:
@duker33 , the name of a class shouldn't describe how it is doing its work. Today it is fetching data and filtering something, tomorrow will fetch prefiltered data, but it will be the same class. Also OOP it is about objects and not about some abstract filters, managers, controllers, etc.

So i believe `TargetCategories` more OOP-related name, unlike the imperative  one `CategoriesFilter`

Up to you

Reply:
@artemiy312 , y, it's good arguments. But in this case we should rename ProductFilter too.
I'll leave as is only not to do additional work. Current name is not the best maybe, but not very awful too